File: src/test/java/com/github/sszuev/graphs/Nodes.java
package com.github.sszuev.graphs;

import org.apache.jena.graph.Node;
import org.apache.jena.graph.NodeFactory;

class Nodes {
    public static Node uri(String uri) {
        return NodeFactory.createURI(uri);
    }

    public static Node bnode() {
        return NodeFactory.createBlankNode();
    }

    public static Node literal(String value) {
        return NodeFactory.createLiteral(value);
    }
}

File: src/test/kotlin/testutils/Nodes.kt
package com.github.sszuev.graphs.testutils

import org.apache.jena.graph.Node
import org.apache.jena.graph.NodeFactory

fun uri(uri: String): Node = NodeFactory.createURI(uri)

fun bnode(): Node = NodeFactory.createBlankNode()

fun literal(value: String): Node = NodeFactory.createLiteral(value)

fun any(): Node = Node.ANY